clang/test/SemaOpenCL/invalid-kernel-attrs.cl

```diff
@@ -32,9 +32,9 @@ void f_kernel_image2d_t( kernel image2d_t image ) { // expected-error {{'kernel'
   int __kernel x; // expected-error {{'__kernel' attribute only applies to functions}}
 }
 
-kernel __attribute__((reqd_work_group_size(1,2,0))) void kernel11(){} // expected-error {{'reqd_work_group_size' attribute requires a positive integral compile time constant expression}}
-kernel __attribute__((reqd_work_group_size(1,0,2))) void kernel12(){} // expected-error {{'reqd_work_group_size' attribute requires a positive integral compile time constant expression}}
-kernel __attribute__((reqd_work_group_size(0,1,2))) void kernel13(){} // expected-error {{'reqd_work_group_size' attribute requires a positive integral compile time constant expression}}
+kernel __attribute__((reqd_work_group_size(1,2,0))) void kernel11(){} // expected-error {{'reqd_work_group_size' attribute must be greater than 0}}
```

Comment:
Should this say something like "**value for** 'reqd_work_group_size' attribute must be greater than 0"?

Reply:
This is a revert back to the behavior from upstream (https://github.com/llvm/llvm-project/blob/main/clang/test/SemaOpenCL/invalid-kernel-attrs.cl#L35) so if we want it reworded it would be better to do upstream.